README.md

```diff
@@ -16,11 +16,15 @@ To install from source:
 
     $ git clone https://github.com/speechmatics/speechmatics-python
     $ cd speechmatics-python && python setup.py install
-    
-or
+
+or with pip:
```

Comment:
This baffles me :) we upload to test.pypi.org but advise installation from source - why don't we recommend installing from test.pypi.org at least?

Reply:
I guess it's because PyPI test is not 'stable'? In any case, this should be released properly to PyPI stable, it has been over a year now.

Reply:
IMO test.pypi.org as a service _is_ stable enough for an "unstable" release ... it's simply a separate instance:

>   You are using TestPyPI – a separate instance of the Python Package Index that allows you to try distribution tools and processes without affecting the real index. 

---

> In any case, this should be released properly to PyPI stable.

Agreed.